jobs/pull_requests.rb

```diff
@@ -8,10 +8,10 @@
 	backend = GithubBackend.new()
 	series = [[],[]]
 	pulls_by_period = backend.pull_count_by_status(
-		:period=>'month', 
-		:orgas=>(ENV['ORGAS'].split(',') if ENV['ORGAS']), 
-		:repos=>(ENV['REPOS'].split(',') if ENV['REPOS']),
-		:since=>ENV['SINCE'],
+		period: 'month',
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.